docs/wiki/Upgrading.md

```diff
@@ -1,24 +1,51 @@
-# Upgrade Process
+# Updating Eventum versions
 
 One of our objectives is to make upgrading from an earlier release as
 painless as possible, and we provide scripts that should bring your
 existing Eventum installation up-to-date.
 
+Eventum supports sequential upgrade path using released minor versions.
+
+In other words, upgrade to minor version separately, you can skip patch
+versions in minor versions.
+
+This table lists upgrade path from SOURCE to TARGET version, the third column
+shows PHP version required for the update.
+
+| SOURCE | TARGET | PHP |
+|--------|--------|-----|
+| 2.2   | 2.3   | 5.1 |
+| 2.3.x | 2.4.0 | 5.1 |
+| 2.4.x | 3.0.0 | 5.1 |
+| 3.0.x | 3.1.0 | 5.3 |
+| 3.1.x | 3.2.0 | 5.5 |
+| 3.2.x | 3.3.0 | 5.6 |
+| 3.3.x | 3.4.0 | 5.6 |
+| 3.4.x | 3.5.0 | 5.6 |
+| 3.5.x | 3.6.0 | 7.1 |
+| 3.6.x | 3.7.0 | 7.1 |
+| 3.7.x | 3.8.0 | 7.1 |
+| 3.8.x | 3.9.0 | 7.2 |
+
+## UTF-8 is required
+
 Please note that if your database encoding is not UTF-8, you may encounter various bugs:
 
 -   PDO Driver is [unusable](https://github.com/eventum/eventum/pull/167)
 -   Issue History entries [get corrupted](https://gitter.im/eventum/eventum?at=58225f1d45c9e3eb4314b58c) (JSON requires UTF-8 encoding)
 
 See 2.2 upgrade instructions how to convert database to UTF-8.
 
-## IMPORTANT
+## Upgrade Process
```

Comment:
IMPORTANT was really good.

Reply:
but it made no sense, it was just an overview of the process. everything on the page is important, otherwise, if it's no important I rather drop the noise :)

Reply:
Ok, may be "Upgrade path"? "Upgrade process" sounds to me like a real running process, command line tool.

Reply:
and "path" sounds to me like a directory, changing to "overview" then